### DISCUSSION TITLE
swap the position of the video windows and list windows

### DISCUSSION THREAD

**Discussion by Conichiwa-Saodeisiga** [SubtitleEdit/subtitleedit#7805] [Ideas]
I'm right-handed, 
and it's a bit weird to have the video window on the right side by the list window, 
and the position of the window can't be fixed after it's split, 
so it's quite awkward to operate. 
Can you guys add a function to swap the video and list windows, pls? 
I'm sure many right-handed users will support my suggestion, 
thanks guys.

**Comment by LeonCheung** [comment]
Update to the latest version, and click the Layout button on the toolbar to explore the magic.
![image](https://github.com/SubtitleEdit/subtitleedit/assets/494788/6b983905-554a-4ef5-9644-01f67d47cc24)

**Reply by Conichiwa-Saodeisiga** [reply]
Good job, waaaaaaaaaaaaoooooooooo... you guys are the best...
